packages/shared/src/components/post/common.tsx

```diff
@@ -5,10 +5,7 @@ import { ArticleOnboardingVersion } from '../../lib/featureValues';
 
 export type PostCardTests = Pick<
   ActionButtonsProps,
-  | 'postCardVersion'
-  | 'postCardShareVersion'
-  | 'postModalByDefault'
-  | 'postEngagementNonClickable'
+  'postCardVersion' | 'postModalByDefault' | 'postEngagementNonClickable'
```

Comment:
Now that I saw this. I think we should create a ticket for cleaning up these ones.

Reply:
yeah, we are awaiting Sab's document on which ones can all be cleanup.
Until then very low priority unless we touch something as with this one 😂